src/java/org/apache/cassandra/cql3/MultiColumnRelation.java

```diff
@@ -95,6 +96,18 @@ public static MultiColumnRelation createInRelation(List<ColumnIdentifier> entiti
         return new MultiColumnRelation(entities, Operator.IN, null, inValues, null);
     }
 
+    /**
+     * Creates a multi-column NOT IN relation with a list of NOT IN values or markers.
+     * For example: {code SELECT ... WHERE (a, b) NOT IN ((0, 1), (2, 3))}
+     *
+     * @param entities the columns on the LHS of the relation
+     * @param inValues a list of Tuples.Literal instances or a Tuples.Raw markers
+     * @return a new <code>MultiColumnRelation</code> instance
+     */public static MultiColumnRelation createNotInRelation(List<ColumnIdentifier> entities, List<? extends Term.MultiColumnRaw> inValues)
```

Comment:
Nit: missing line break
```suggestion
     */
     public static MultiColumnRelation createNotInRelation(List<ColumnIdentifier> entities, List<? extends Term.MultiColumnRaw> inValues)
```